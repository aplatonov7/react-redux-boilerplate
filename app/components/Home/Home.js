import React, { PropTypes } from 'react'
import s from './Home.scss'

const Home = () => (
  <div className={s.root}>
    Yep, it works
  </div>
)

Home.propTypes = {

}

export default Home
