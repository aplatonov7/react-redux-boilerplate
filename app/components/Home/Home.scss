@import 'variables';
@import 'mixins';

.root {

}
