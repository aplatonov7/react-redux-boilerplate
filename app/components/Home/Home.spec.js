import React from 'react'
import renderer from 'react-test-renderer'
import Home from './'

describe('Home component', () => {
  const item = renderer.create(<Home />)

  it('should match snapshot', () => {
    expect(item.toJSON()).toMatchSnapshot()
  })
})
