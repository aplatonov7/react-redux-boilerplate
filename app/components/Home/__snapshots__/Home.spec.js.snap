exports[`Home component should match snapshot 1`] = `
<div
  className="root">
  Yep, it works
</div>
`;
